packages/api/tests/endpoints/test-rules.js

```diff
@@ -231,6 +236,14 @@ test('CUMULUS-911 POST with pathParameters and without an Authorization header r
   assertions.isAuthorizationMissingResponse(t, response);
 });
 
+test('CUMULUS-911 PATCH with pathParameters and without an Authorization header returns an Authorization Missing response', async (t) => {
+  const response = await request(app)
+    .patch('/rules/asdf')
+    .set('Accept', 'application/json')
+    .expect(401);
```

Comment:
Do we not need to set the Cumulus-API-Version?

Reply:
`const { version } = require('../../lib/version');` adds 'cumulus-api-version'

Reply:
Ahh thanks for the explanation!